README.md

```diff
@@ -103,23 +108,37 @@ Note: These instructions have primarily been tested for Mac/Linux/WSL environmen
 
 3. Create an IoT TwinMaker workspace.
 
-   Different Digital Twin applications use different resources. Run the following command to create a role for our workspace that has the necessary permissions for this sample application. Note that you will use the role name in the output in the next step.
+   a. Create a Grafana dashboard IAM role
```

Comment:
Should workspace role creation still be first as it's required for workspace

Maybe the following?

a. TwinMaker execution role
b. Grafana role
c. Create workspace

Reply:
Updated the README to reflect this